tests/test_optim/test_scheduler/test_param_scheduler.py

```diff
@@ -438,6 +438,24 @@ def test_cosine_restart_scheduler(self):
             eta_min=0)
         self._test_scheduler_value(scheduler, targets, epochs=10)
 
+        epochs = 10
+        t = 10
+        eta_min = 1e-7
```

Comment:
The absolute tolerance given in `_test_scheduler_value` is `1e-5`, `1e-7` is too small to be recognized as an error.